tests/core/gui/test_revoked_user.py

```diff
@@ -2,31 +2,15 @@
 
 import pendulum
 import pytest
-from PyQt5 import QtCore
 from PyQt5.QtWidgets import QLabel
 
 from parsec.api.data import RevokedUserCertificateContent
-from parsec.core.local_device import save_device_with_password
-
-
-@pytest.fixture
-async def logged_gui(aqtbot, gui_factory, running_backend, autoclose_dialog, core_config, bob):
-    save_device_with_password(core_config.config_dir, bob, "P@ssw0rd")
-    gui = await gui_factory()
-    lw = gui.test_get_login_widget()
-    tabw = gui.test_get_tab()
-
-    # assert lw.combo_username.currentText() == f"{bob.organization_id}:{bob.device_id}"
-    await aqtbot.key_clicks(lw.line_edit_password, "P@ssw0rd")
-
-    async with aqtbot.wait_signals([lw.login_with_password_clicked, tabw.logged_in]):
-        await aqtbot.mouse_click(lw.button_login, QtCore.Qt.LeftButton)
-
-    return gui
+from tests.common import customize_fixtures
 
 
 @pytest.mark.gui
 @pytest.mark.trio
+@customize_fixtures(logged_gui_as_admin=False)
```

Comment:
no need to set this to false given it's the value by default ;-)

Reply:
Can we keep it to explicitly point that we are logged as bob ?